src/main/java/mateacademy/onlinebookstore/dto/category/CategoryRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package mateacademy.onlinebookstore.dto.category;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CategoryRequestDto {
+    @NotNull
```

Comment:
NotBlank

Reply:
Also, it can be record ;)

Reply:
but I thought when I need to use some constraints (the annotation below fields) better use Dto not record...